Add missing page description metadata (#260)
hashicorp/terraform-docs-common: website/docs/cloud-docs/api-docs/feature-sets.mdx

---
page_title: Feature Sets - API Docs - Terraform Cloud
tfc_only: true
description: >-

>   Use the `/feature-sets` endpoint to interact with feature sets. List feature sets, and those eligible to an organization, using the HTTP API.

Reviewer: ```suggestion
  Use the `/feature-sets` endpoint to review with feature sets. List available feature sets and which feature sets an organization is eligible for using the HTTP API.
```
